sdk/anomalydetector/azure-ai-anomalydetector/src/test/java/com/azure/ai/anomalydetector/MultivariateAnomalyDetectorClientTest.java

```diff
@@ -3,37 +3,162 @@
 
 package com.azure.ai.anomalydetector;
 
-import com.azure.ai.anomalydetector.models.AnomalyDetectionModel;
+import com.azure.ai.anomalydetector.models.*;
```

Comment:
Don't do this `import *`. Configure your IDE to avoid that.

Reply:
fixed it.